LoopKit/CarbKit/CarbStore.swift

```diff
@@ -1294,6 +1294,86 @@ extension CarbStore {
     }
 }
 
+// MARK: - Critical Event Log Export
+
+extension CarbStore: CriticalEventLog {
```

Comment:
Agree with Rick; if this work is really isolated to an extension, putting it into a separate file might make this a little more navigable.

Reply:
Okay, I can do this, but if we are going this way, we should make this a general, well-known policy. We haven't been doing this before.

Reply:
I don't feel that strongly, though, FWIW

Reply:
Yeah, it's not a big issue for me either; just seems a little more navigable.

Reply:
I'd be fine if we don't do this here.

Reply:
The code in this extension uses the managed object context, which is a private property. To move this extension out of the main file would require changing the property access from private to internal. I believe keeping the property private is more important than moving the extension to a separate file. Going to leave as-is unless anyone objects.

Reply:
✅